src/plugins/data/public/query/query_string/dataset_service/dataset_service.ts

```diff
@@ -164,6 +165,18 @@ export class DatasetService {
     });
   }
 
+  public refreshCache(): void {
+    this.sessionStorage.clear();
+  }
+
+  public getLastCacheTime(): string | undefined {
```

Comment:
want to convert to a Date given its passed a date? otherwise consumers just need to handle this

Reply:
yeah we should make sure to do this.

@sejli 
we should probably include the day and should also pulling from UI Settings to get the format of the date
<img width="929" alt="Screenshot 2024-09-18 at 3 37 48 PM" src="https://github.com/user-attachments/assets/40c46ec9-105e-42b9-925c-18c5d40cbad2">